docs/packaging/packaging-with-visual-studio.md

```diff
@@ -0,0 +1,81 @@
+## Overview
+This page will cover everything needed to package an application into an isolated Win32 app. 
+
+## Required software pre-requisites:
+* **Visual Studio** version 17.10.2 or higher
+
+## Step 1 - Install the required workloads in Visual Studio including Windows 11 SDK 10.0.26100.0 (or greater)
+Open the Visual Studio Installer and Modify the existing installation. 
+On the Workloads tab check the "Windows application development" box for development in C# 
+(and optionally check the "C++ WinUI app development tools" box for development in C++). 
+The Windows 11 SDK 10.0.26100.0 (or greater) is also required.
+
+![image](images/vs/01-VisualStudio-Required-Workloads.png)
+
+## Step 2 - Create your app project
+In Visual Studio, create a new C# or C++ project using the template *Blank App, 
+Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)*. 
+Click "Create" and then select "10.0.26100.0" (or later) for Target Platform Version (TPV) 
+and Target Platform Minimum Version (TPMinV). 
+
+![image](images/vs/02-Blank-App-Packaged-With-Windows-Application-Packaging-Project.png)
+![image](images/vs/03-New-Project.png)
+![image](images/vs/04-New-Windows-Project.png)
+
+## Step 3 - Set NuGet to use Microsoft.Windows.SDK.BuildTools version 10.0.26100.1 or later
+Go to Project -> "Manage NuGet Packages" to install Microsoft.Windows.SDK.BuildTools 
+version 10.0.26100.1 (or later).
```

Comment:
First of all, this falls into the category of giant images.

I'm still confused about the wording. What's the purpose of this step? The title says set `NuGet` to use the SDK. The instructions seem to install the SDK. We have installed the SDK previously in the very first step. Is this something extra?

Reply:
NuGet is a package manager. The developer needs to configure the build tools used to use Microsoft.Windows.SDK.BuildTools version 10.0.26100.1. Just having the version set does not use the version so this step is required and included.

Reply:
Why is the package manager needed to configure build tools?

Reply:
Q: Why are build tools needed? A: Build tools are used with the compile/publish options are completed.
Q: Why is the package manager needed to configure build tools? A: This is the place where build tools are configured. It's a NuGet package so you go to the NuGet configuration menu.